### PR TITLE
Docs: Add missing tabs in Create a Batch Job Strategy

### PR DESCRIPTION
**Pull request by akarsh-jain-790**
- What - Fixes:  #2423

- Why - Tabs separating the different methods of sending a request needed to be added.

- How - Following the guideline in the issue made the changes to medusa/docs/content/usage/configurations.md

- Testing - No test written because it's a markup file.

### CODE REVIEW

File: docs/content/advanced/backend/batch-jobs/create.md [hunk 1]
```diff
@@ -251,6 +254,9 @@ The first step is to create a batch job using the [Create Batch Job endpoint](ht
 
 For example, this creates a batch job of the type `publish-products`:
 
+<Tabs groupId="request-types">
+<TabItem value="client" label="Medusa JS Client" default>
+
 ```jsx
 // using JS Client
```
Comment (shahednasser) [inline docs/content/advanced/backend/batch-jobs/create.md:261]: There's a line similar to this for all blocks can you remove them? For JS Client, Fetch API, and cURL code snippets.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your contribution @akarsh-jain-790 I just have a small comment.

File: docs/content/advanced/backend/batch-jobs/create.md (hunk 1 above)
Comment (akarsh-jain-790) [reply docs/content/advanced/backend/batch-jobs/create.md:261]: If I understood you correctly, you would like me to remove the blank line?

Comment (shahednasser) [reply docs/content/advanced/backend/batch-jobs/create.md:261]: No I meant the `//using JS Client` line. Every code block for JS Client, Fetch API, and cURL have a line at the beginning similar to this one. They should be removed. Sorry if it wasn't clear.

**Comment by akarsh-jain-790** [comment]
@shahednasser Have implemented your suggested changes, please let me know if there is anything else I can improve!

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Thanks! @akarsh-jain-790  🚀

**Pull request merged**